VMBackup/main/backuplogger.py

```diff
@@ -60,25 +65,33 @@ def log_to_con(self, msg):
                 C.write(message.encode('ascii','ignore'))
         except IOError as e:
             pass
+        except Exception as e:
+            log_msg = "Excetion in log_to_con_py3"
+            C.write(log_msg)
```

Comment:
have you tested this flow. "C" seems to be scoped in the with block.
It seems shouldn't work.

Reply:
Good catch. I had not tested this removing this from here.